### DISCUSSION TITLE
Remove dependency of unpkg.com

### DISCUSSION THREAD

**Discussion by d-shannon** [Kyon147/laravel-shopify#138] [Ideas]
Hey all,

As some of you may have noticed that [unpkg.com/](https://unpkg.com/) was down for an extended period of time. Any thoughts on moving away from the unpkg cdn and adding it as an npm dependancy?

On a side note, I see that the shopify "app-bridge-utils" npm package has been depecriated - see below:

https://www.npmjs.com/package/@shopify/app-bridge-utils 

Does https://www.npmjs.com/package/@shopify/app-bridge incorporate all we need?

Many thanks

**Comment by kurakin-oleksandr** [comment]
Actually, continue using of `app-bridge-utils` blocks upgrading app bridge version higher than 3.5.1 (current version is already 3.7.5)

**Comment by Kyon147** [comment]
There's a task to allow us to control the package manager  - so we might as well add in the removal of the `app-bridge-utils` if this is no longer needed.
